docs/guides/ide/editcl.md

```diff
@@ -1,20 +1,39 @@
 # Editing CL
 
-Apart from those common language features mentioned in [Edit](edit.md), CL source editing also has these capabilities:
+Apart from those common language features mentioned in [Edit](edit.md), CL source editing also has the following capabilities:
```

Comment:
Seems like this edit.md file cannot be referenced. Need to change the link to be `./guides/ide/edit.md`. The other places may also have the same issue.

Reply:
Thanks for catching that